src/sentry/conf/server.py

```diff
@@ -881,6 +881,7 @@ def create_partitioned_queues(name):
     "TEST_REQUEST_DEFAULT_FORMAT": "json",
     "DEFAULT_PERMISSION_CLASSES": ("sentry.api.permissions.NoPermission",),
     "EXCEPTION_HANDLER": "sentry.api.handlers.custom_exception_handler",
+    "DEFAULT_VERSIONING_CLASS": "rest_framework.versioning.QueryParameterVersioning",
```

Comment:
Do we need to set `ALLOWED_VERSIONS` in settings?

Reply:
The docs seem to be a little terse on that setting. AFAICT, it would be a closed set of allowed values, like `(1, 2, 3)`? In that case, I think it would apply only to a global set of API versions. But we're having each individual endpoint define which versions exist for it.